docs/1.2-DRAFT/context.jsonld

```diff
@@ -2892,6 +2895,16 @@
           "ResourceDescriptor": "http://www.w3.org/ns/dx/prof/ResourceDescriptor",
           "ResourceRole": "http://www.w3.org/ns/dx/prof/ResourceRole",
           "Profile": "http://www.w3.org/ns/dx/prof/Profile",
+          "softwareSuggestions": "https://w3id.org/codemeta#softwareSuggestions",
+          "continuousIntegration": "https://w3id.org/codemeta#continuousIntegration",
+          "buildInstructions": "https://w3id.org/codemeta#buildInstructions",
+          "developmentStatus": "https://w3id.org/codemeta#developmentStatus",
+          "embargoEndDate": "https://w3id.org/codemeta#embargoEndDate",
+          "readme": "https://w3id.org/codemeta#readme",
+          "issueTracker": "https://w3id.org/codemeta#issueTracker",
+          "referencePublication": "https://w3id.org/codemeta#referencePublication",
+          "hasSourceCode": "https://w3id.org/codemeta#hasSourceCode",
+          "isSourceCodeOf": "https://w3id.org/codemeta#isSourceCodeOf",
```

Comment:
As of today, the identifier is https://codemeta.github.io/terms/

I have been pushing to shift it to https://w3id.org/codemeta/, which is where we currently have the contexts hosted. But it's not yet the case.